### DISCUSSION TITLE
How to do computed properties?

### DISCUSSION THREAD

**Discussion by wobsoriano** [alpinejs/alpine#507] [4. General]
In Vue 2, we can do something like

```javascript
new Vue({
    data: () => ({
      items: []
    }),
    computed: {
      computedItems() {
        return this.items.map(i => ({
          ...i,
          anotherProperty: 'hello-world'
        }))
      }
    }
})
```

How can we do this in Alpine?

**Comment by ryangjchandler** [comment]
Hey @sorxrob, I wrote a couple of articles on computed properties:

https://ryangjchandler.co.uk/articles/pseudo-computed-properties-in-alpine

https://ryangjchandler.co.uk/articles/an-alternative-approach-to-computed-properties-in-alpinejs

**Reply by wobsoriano** [reply]
> Hey @sorxrob, I wrote a couple of articles on computed properties:
> 
> https://ryangjchandler.co.uk/articles/pseudo-computed-properties-in-alpine
> 
> https://ryangjchandler.co.uk/articles/an-alternative-approach-to-computed-properties-in-alpinejs

Thanks! So I was actually doing it right then (first article). Awesome articles.
